platforms/software/dependency-management/src/main/java/org/gradle/api/internal/artifacts/ivyservice/moduleconverter/dependencies/DefaultLocalConfigurationMetadataBuilder.java

```diff
@@ -151,33 +140,31 @@ public void visitChildVariant(String name, DisplayName displayName, ImmutableAtt
     }
 
     private static CalculatedValue<ImmutableList<LocalComponentArtifactMetadata>> getConfigurationArtifacts(
-        LocalComponentMetadata parent, ModelContainer<?> model, CalculatedValueContainerFactory calculatedValueContainerFactory,
-        String name, String description, ImmutableSet<String> hierarchy, List<PublishArtifact> sourceArtifacts
+        ModelContainer<?> model, CalculatedValueContainerFactory calculatedValueContainerFactory,
+        String name, String description, ImmutableSet<String> hierarchy, ConfigurationsProvider configurationsProvider, ComponentIdentifier componentId
```

Comment:
💅 could you maybe reorder these parameters to group them by "type" (services vs model vs inputs?)